relay-aws-extension/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "relay-aws-extension"
```

Comment:
Could you please also reference this new crate from Relay's root docs? That needs two entries:

https://github.com/getsentry/relay/blob/0a4a2b1ed6da1e4a47383e975ece2fc8e14c475b/relay/src/main.rs#L28-L30

and the reference at the bottom:

https://github.com/getsentry/relay/blob/0a4a2b1ed6da1e4a47383e975ece2fc8e14c475b/relay/src/main.rs#L63